src/FirstSteps/RPG/Equipment.cs

```diff
@@ -0,0 +1,78 @@
+﻿using FirstSteps.RPG.Items;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using FirstSteps.RPG.Heroes;
+
+namespace FirstSteps.RPG
+{
+    public class Equipment
+    {
```

Comment:
You do not stick to naming conventions.
`const` - start from uppercase ex. MaxWeight
`private fields` - start from _ ex. _backPack
`public fields` - start from uppercase but it's better to use properties in that cases